tests/docker-scripts/docker-compose.yml

```diff
@@ -24,15 +41,15 @@ services:
       - ../../tests:/work/tests
       - ../../test-results:/work/test-results
       - ../../playwright-report:/work/playwright-report
-    entrypoint: 'npm run e2e -- --config=playwright-ci.config.ts'
+    entrypoint: '/bin/bash --login -i -c "npm run e2e -- --config=playwright-ci.config.ts"'
     depends_on:
       dhc-server:
         condition: service_healthy
 
   web-ui-update-snapshots:
     extends:
       service: web-ui-tests
-    entrypoint: 'npm run e2e:update-snapshots -- --config=playwright-ci.config.ts'
+    entrypoint: '/bin/bash --login -i -c "npm run e2e:update-snapshots -- --config=playwright-ci.config.ts"'
```

Comment:
It seems we can no longer pass in additional args to constrain this command e.g. before I could

```sh
npm run e2e:update-ci-snapshots styleguide
```
or 

```sh
npm run e2e:update-ci-snapshots -- -g "some test name"
```

Not sure if this is the culprit, but it doesn't seem to be working anymore

Reply:
This is the culprit. The params just get passed to the entrypoint, so now it's passing those params at the end of the bash command instead of in the npm command.

Might be fixed by adding $@ to the end of the npm command in the quotes

Reply:
I've tried this a few different ways but can't get the $@ args to get included